packages/cli/src/commands/validator/status.ts

```diff
@@ -0,0 +1,117 @@
+import { Address } from '@celo/contractkit'
+import { flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import { range } from 'lodash'
+import * as rlp from 'rlp'
+import { Block } from 'web3/eth/types'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { Flags } from '../../utils/command'
+
+export default class ValidatorOnline extends BaseCommand {
+  static description =
+    'Show information about whether the validator signer is elected and validating. This command will check that the validator meets the registration requirements, and its signer is currently elected and actively signing blocks.'
+
+  static flags = {
+    ...BaseCommand.flags,
+    signer: Flags.address({
```

Comment:
I want this command to work for signers that are no longer authorized in the smart contract, but are still elected for the epoch. Such as when someone has changed their keys and is waiting for the next epoch to come while running two validators, one with the old signing key and one with the new. I figure it will be helpful to know when your old validator is officially safe to take offline.